Userland/Libraries/LibWeb/Loader/LoadRequest.h

```diff
@@ -37,6 +37,9 @@ class LoadRequest {
     void start_timer() { m_load_timer.start(); };
     Time load_time() const { return m_load_timer.elapsed_time(); }
 
+    auto page() const { return m_page; };
```

Comment:
Not sure about using `auto` for the return type of methods, it's not usually done in Serenity...
```suggestion
    Optional<Page*> page() const { return m_page; }
```